### DISCUSSION TITLE
Warning when using run-compute()

### DISCUSSION THREAD

**Discussion by minnieliao** [phoebe-project/phoebe2#805] [Questions]
Hi, I would like to inquire if anyone else is experiencing this issue as well. I'm unsure whether it may be attributed to the wrong version,  but even after upgrading to the latest version, it still persists.
Below is my code.
```
import phoebe
import numpy as np
%matplotlib inline

cb =phoebe.default_binary(contact_binary =True)

#temperature
cb['teff@primary'] = 6000
cb['teff@secondary'] =6200

#mass ratio Mp/Ms
cb['q@binary'] =0.8

#inclnation
cb['incl'] =0.85

#period
cb['period@orbit'] =0.8

#light curve
cb.add_dataset('lc', compute_times =phoebe.linspace(0, 1, 51), dataset ='lc01', overwrite =True)

cb.run_compute()
```
It shows warning like this.
**WARNING: File may have been truncated: actual file length (212992) is smaller than the expected size (2160000) [astropy.io.fits.file]**

Thanks.

**Comment by kecnry** [comment]
I suspect this is coming from the internal passband files - the download must have been interrupted.  You should be able to force redownloading it by either deleting the file locally, or by calling [`download_passband('Johnson:V')`](http://phoebe-project.org/docs/2.4/api/phoebe.atmospheres.passbands.download_passband).  Please do let us know if that works!

**Reply by minnieliao** [reply]
It works!
